web/static/web/css/footer.css

```diff
@@ -91,3 +91,7 @@
         grid-column-start: auto;
     }
 }
+
+#footer .content {
+    text-align: right;
+}
```

Comment:
This should be moved to a `@media screen and (min-width: 993px)` block, so that it doesn't apply on mobile layout 🙂 A suggestion is to place that block before the `max-width: 992px` block, to be consistent with e.g. `header.css`.

If you want, you could also add a comment above the `max-width: 992px` block saying something like `/* Switch to mobile layout */`, which would make the CSS file more readable 😊

Lastly, remember to include a newline at the end of a file 😬 It doesn't make any difference in practice, but it's a convention 😅

Reply:
On second thought - after having looked at the other contents of this file, it's probably better to simply move the `text-align: right;` property to the `#footer` block at line 5 😁

Reply:
On second thought - after having looked at the rest of the contents of this file, it's probably better to instead add this block to the `@media screen and (max-width: 992px)` block:
```
#footer .content {
    text-align: center;
}
```
And also move the existing `#footer .content` block to just below e.g. the `#footer .social.list i` block (both to let blocks further down the file overwrite the properties of the blocks above, and to group blocks affecting neighboring elements).